lib/oxidized/model/tplink.rb

```diff
@@ -18,13 +18,17 @@ class TPLink < Oxidized::Model
   end
 
   cmd :all do |cfg|
+    # remove unwanted paging line
+    cfg.gsub! /Press any key to contine.*/, ''
     # normalize linefeeds
     cfg.gsub! /(\r|\r\n|\n\r)/, "\n"
     # remove empty lines
     cfg.each_line.reject { |line| line.match /^[\r\n\s\u0000#]+$/ }.join
   end
 
   cmd :secret do |cfg|
+    cfg.gsub! /enable password (\S+)/, 'enable password <secret hidden>'
```

Comment:
Do these statements always appear the the beginning of the line, or are they part of other commands?
Could we prefix them with ^ to ensure we don't catch these words appearing elsewhere?

Reply:
I cant say this for all models, that's why I initially did not wrote a better regex, i.e using '^'. At least for the tree different models I own, the term 'enable password (\S+)' is not part of other commands.
However, the regex `/^enable password (\S+)/` should work too and is more specific, so I will try it.

Reply:
After review: yes, its possible to use this regex. Other firmware versions use a different approach (`enable secret ....` in Firmware from 2014, encrypted output) (`enale secret` in firmware starting from 2015, encrypted output), so its fine to use the better regex.